### PR TITLE
[Release] Add release logs for 1.11.0rc0 (GCS KV & pubsub not enabled)

### PR DESCRIPTION
**Pull request by mwtian**
## Why are these changes needed?
Before we build 1.11.0rc1 with GCS HA enabled, record current benchmark results. This makes releasing 1.11.0 without GCS-Ray possible, and helps to detect performance regression from enabling GCS-Ray.

For comparing against the result in 1.10.0, check the b3f64ff1b5b6b43bcd7e016849651f52a5aef200 commit only.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "success": 1,
-  "avg_pg_create_time_ms": 0.6551887567598629,
-  "avg_pg_remove_time_ms": 3.2382229564551754,
-  "_runtime": 294.8811585903168,
-  "_session_url": "https://beta.anyscale.com/o/anyscale-internal/projects/prj_2xR6uT6t7jJuu1aCwWMsle/clusters/ses_JrxFXTCkW1yA9mnRZKTayGEe",
-  "_commit_url": "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/1.10.0/5ea565317a8104c04ae7892bb9bb41c6d72f12df/ray-1.10.0-cp37-cp37m-manylinux2014_x86_64.whl",
+  "avg_pg_create_time_ms": 1.246994148648686,
```
Comment (mwtian) [inline release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json:3]: Is this regression or noise? @rkooo567

File: release/release_logs/1.11.0/microbenchmark.json [hunk 2]
```diff
@@ -1,138 +1,126 @@
...
-    366.82602393524735
-  ],
-  "multi_client_put_calls": [
-    189007.7322366993,
-    747.0416719411013
-  ],
   "single_client_get_calls_Plasma_Store": [
-    5822.640574716487,
-    139.29057092881047
+    4417.483407539963,
```
Comment (mwtian) [inline release/release_logs/1.11.0/microbenchmark.json:3]: Is this a regression? cc @rkooo567 @scv119

File: release/release_logs/1.11.0/microbenchmark.json [hunk 3]
```diff
@@ -1,138 +1,126 @@
 {
-  "single_client_get_calls": [
```
Comment (mwtian) [inline release/release_logs/1.11.0/microbenchmark.json]: Looking into why these are missing.

File: release/release_logs/1.11.0/microbenchmark.json (hunk 2 above)
Comment (scv119) [reply release/release_logs/1.11.0/microbenchmark.json:3]: yup I think this is a regression.

File: release/release_logs/1.11.0/microbenchmark.json (hunk 3 above)
Comment (mwtian) [reply release/release_logs/1.11.0/microbenchmark.json]: These microbenchmarks are removed here: https://github.com/ray-project/ray/pull/21284#discussion_r779159892

File: release/release_logs/1.11.0/microbenchmark.json (hunk 2 above)
Comment (rkooo567) [reply release/release_logs/1.11.0/microbenchmark.json:3]:
```
  "single_client_put_gigabytes": [
    9.203941431281354,
    7.382235276551273
  ],
```

This was 20GB in 1.10, so I think there was a regression in plasma store

File: release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json (hunk 1 above)
Comment (rkooo567) [reply release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json:3]: Screen Shot 2022-02-01 at 6 20 49 PM
^ Lower is better

I think it is noise. I found the variance is pretty big (0.7~1.1). Do you mind running this one more time?

Comment (mwtian) [reply release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json:3]: Reran and it was 1.15: https://buildkite.com/ray-project/periodic-ci/builds/2610#5932b008-a14f-46a5-91b7-654bc5b7141d/113-388

**Comment by mwtian** [comment]
All regressions are determined to be no longer P0. This is mergeable.

**Review by ericl — APPROVED** [review]

**Pull request merged**